src/iexamine.cpp

```diff
@@ -1172,7 +1172,7 @@ void iexamine::flower_poppy(player *p, map *m, int examx, int examy)
     m->spawn_item(examx, examy, "poppy_bud");
 }
 
-void iexamine::flower_blubell(player *p, map *m, int examx, int examy)
+void iexamine::flower_bluebell(player *p, map *m, int examx, int examy)
```

Comment:
The current handoff-setup expressly looks for the non-e spelling.  After reverting the hash stuff, if you keep the speling change you'll need to change it there too.